client/driver/executor/executor.go

```diff
@@ -252,8 +252,8 @@ func (e *UniversalExecutor) LaunchCmd(command *ExecCommand) (*ProcessState, erro
 	if err := e.configureLoggers(); err != nil {
 		return nil, err
 	}
-	e.cmd.Stdout = e.lro
-	e.cmd.Stderr = e.lre
+	e.cmd.Stdout = e.lro.processOutWriter
+	e.cmd.Stderr = e.lre.processOutWriter
```

Comment:
should this be processErrWriter?

Reply:
No that is just the writer end of the pipe. The difference is `e.lro` and `e.lre` which are log rotator output and error respectively